h2o-docs/src/product/data-science/algo-params/distribution.rst

```diff
@@ -80,6 +80,87 @@ Some examples of response distributions are provided below.
+| Modified_Huber    | - with :math:`yf=(2y-1)*f`                                                                |
+|                   | - :math:`yf<-1`:                                                                          |
+|                   | - :math:`-4w*(yf)`                                                                        |
+|                   | - :math:`yf>1` : :math:`0`                                                                |
+|                   | - else: :math:`w*(yf)^2`                                                                  |
++-------------------+-------------------------------------------------------------------------------------------+
+| Tweedie           | with :math:`f=\text{link}(f)`, requires :math:`1>\text{Tweedie_Power}<2`:                 |
+|                   | :math:`2w*y^{(2-\text{Tweedie_Power})/((1-\text{Tweedie_Power})*(2-\text{Tweedie_Power})) |
+|                   | -y*\text{exp}(f*(1-\text{Tweedie_Power}))/(1-\text{Tweedie_Power})                        |
+|                   | +\text{exp}(f*(2-\text{Tweedie_Power}))/(2-\text{Tweedie_Power})}`                        |
++-------------------+-------------------------------------------------------------------------------------------+
+
++-------------------+---------------------------------------------------------------------------+
+| Link/Inverse Link | Equation                                                                  |
+| Functions         |                                                                           |
++===================+===========================================================================+
+| Identity: Gaussian| - :math:`link(f)=f`                                                       |
+| Huber, Laplace,   | - :math:`linkInversion(f)=f`                                              |
+| Quantile          |                                                                           |
++-------------------+---------------------------------------------------------------------------+
+| Log: Multinomial, | - :math:`link(f)=log(f)`                                                  |
+| Poisson, Gamma,   | - :math:`linkInversion(f)=exp(f)`                                         |
+| Tweedie           |                                                                           |
++-------------------+---------------------------------------------------------------------------+
+| Logit: Bernoulli, | - :math:`link(f)=(f/(f-1))`                                               |
+| Quasibinomial,    | - :math:`linkInversion(f)=1/(1+exp(-f))`                                  |
+| Modified_Huber    |                                                                           |
++-------------------+---------------------------------------------------------------------------+
+| Inversion         | - :math:`link(f) = 1/f (where f = f < 0 ? min(-1e-5, f) : max(-1e-5, f))` |
```

Comment:
I think most users won't be familiar with Java syntax. 

I would reformat it to:
(where if f < 0: f = min(-1e-5, f), else: f = max(-1e-5, f))
@maurever , am I interpreting this right?

Reply:
Yes, you are right, @neemah2o, the syntax is not readable. I just rewrite it from code to help @hannah-tillman to interpret it into the doc. There is also the possibility to use the `cases` formula:

```
.. math::
    f =
    \begin{cases}
        min(-1e-5, f) & \text{for }f<0 \\
        max(-1e-5, f)& \text{for }f>0 \\
    \end{cases}
```